Sources/generate-enum-properties/main.swift

```diff
@@ -27,6 +27,9 @@ usage: \(command) [--help|-h] [--dry-run|-n] [<file>...]
     -n, --dry-run
         Don't update files in place. Print to stdout instead.
 
+    --include-plain-enums
```

Comment:
could we rename this option to `--include-raw-enums` to better signify that this is dealing with enums that have no associated data amongst all their cases?

Reply:
as you can tell Ive had trouble naming this. In my mind, "raw" enum means one that is RawRepresentable, which isn't necessarily the case here. But if you feel better about that naming then Im all for it. Will push that change when I get to my mac.

Reply:
We’re definitely up to workshop things further. Our impression was “plain enum” doesn’t have a community meaning, so maybe we could do something more explicitly descriptive, like `--include-all` with help text to explain that this will include all enums, even those without associated values.

Reply:
`--include-all` nice and simple, love it